src/org/usfirst/frc/team2585/systems/WheelSystem.java

```diff
@@ -94,25 +128,71 @@ public double rampedForward() {
 		}
 		return forward;
 	}
+	
+	/**
+	 * @param rotation the input rotation value
+	 * @return the result of applying the exponent to the ROTATION_EXPONENT while maintaining its sign
+	 */
+	public double applyRotationExponent(double rotation) {
+		boolean negRotation = false;
+		if (rotation < 0) {
```

Comment:
You can use Math.signum to calculate the sign of rotation. This will help the readability of this function since you just need to multiply the result of your pow by it.